src/commands/dev/edge/mod.rs

```diff
@@ -34,7 +35,11 @@ pub fn dev(
     let runtime = TokioRuntime::new()?;
     loop {
         let target = target.clone();
-        let user = user.clone();
+        let mut user = user.clone();
+
+        // Check if oauth token is expired
+        let _res = check_update_oauth_token(&mut user).expect("Failed to refresh access token");
```

Comment:
While it isn't likely that the user will be able to do anything if the `expect` is reached from an Err value, we may want to consider letting the `dev` caller handle this error or print something more useful here. `"Failed to refresh access token"` isn't as informative as whatever might come back from the API for this call. 

@ocsfrank could you dig in a bit and see what other options are available here? The end-goal is providing as much to the `wrangler` user as possible for debugging.. as this stands, we just print this panic message and maybe a backtrace before suggesting they `wrangler report` the issue. But, it's unlikely that we can help them, or that they can help themselves without more info.